rust/mxl/src/flow/reader.rs

```diff
@@ -24,7 +24,7 @@ pub(crate) fn get_flow_info(
 ) -> Result<FlowInfo> {
     let mut flow_info: mxl_sys::mxlFlowInfo = unsafe { std::mem::zeroed() };
     unsafe {
-        Error::from_status(context.api.flow_reader_get_info(reader, &mut flow_info))?;
+        Error::from_status(context.api.mxlFlowReaderGetInfo(reader, &mut flow_info))?;
```

Comment:
Do you want the change from snake case to camel case?

Looks like a bindgen ParseCallbacks could be used to auto generate snake case if not?

Reply:
Good idea. I didn't bother too much because it's only on the sys crate, so not visible to the regular end user, and I didn't want to manually annotate fields, because the whole purpose of the PR is to automate the process. If it can be automated, I'm all for it. I'll give it a try later today. Thanks!

Reply:
Done.

Reply:
Pretty cool right? Even if it actually makes for a bigger diff?